docs/content.go

```diff
@@ -230,6 +230,31 @@ Enterprise Performance Computing (EPC)`
 	KeyImportExample string = `
   $ singularity key import $HOME/key.asc
   $ singularity keys import $HOME/key.asc
+  `
+
+	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	// key export
+	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	KeyExportUse   string = `export [export options...] <key-fingerprint> <full-path-to-key-file>`
```

Comment:
the last argument should be described as "output file" to make it clear that this is the file that will be written.